index.bs

```diff
@@ -2493,11 +2538,12 @@ To <dfn>obtain and deliver a debug report on source registration</dfn> given a
 
 1. If |dataType| is:
     <dl class="switch">
-    : "<code>[=source debug data type/source-destination-limit=]</code>"
-    :: [=map/Set=] |body|["`limit`"] to the user agent's [=max destinations covered by unexpired sources=],
-         [=serialize an integer|serialized=].
+    : "<code>[=source debug data type/source-success=]</code>"
+    : "<code>[=source debug data type/source-noised=]</code>"
+    :: If |destinationLimitHit| is true, [=map/set=] |body|["`source_destination_limit`"] to the user agent's [=max destinations covered by unexpired sources=],
```

Comment:
Please add a non-normative note to explain why we're adding this field when the debug data type is `success` or `noised`.

Reply:
Added, thanks.